string frame_id
float64 stamp
float64 x_pos
float64 y_pos
float64 z_pos
float64 x_ori
float64 y_ori
float64 z_ori
float64 w_ori
float64 x_vel
float64 y_vel
float64 z_vel
float64 x_acc
float64 y_acc
float64 z_acc
